src/sqlfluff/rules/L016.py

```diff
@@ -369,6 +369,34 @@ def _gen_line_so_far(raw_stack, initial_buff=None):
                 break  # pragma: no cover
```

Comment:
Not related to your change, but since you're in the code here anyway, is there a test case we should add for this?

Reply:
I think I'll pass on adding the test cases. Some of these rules are so large and complex, you can end up fixing an issue while gaining little or no insight into other areas. 😢

Reply:
Can't blame me for trying! 😉 Resolving.